strawberry_django_plus/type.py

```diff
@@ -89,13 +118,21 @@ def _from_django_type(
 
         field = cast(StrawberryDjangoField, field)
 
-        # FIXME: Improve this...
         if not field.base_resolver:
 
-            def conn_resolver(root):
-                return getattr(root, name).all()
+            def conn_resolver(parent_annotation: Optional[StrawberryAnnotation], root, info: Info):
```

Comment:
Can you add a test with a nested connection to make sure that this is working as expected and without any regressions?

Reply:
Added a test and found an method to access custom querysets on related models. Just has to be specified in the Meta class of a model's class

Reply:
`tests/test_optimizer.py:test_query_connection_nested` was also useful in catching a potential regression :+1: